javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,39 @@
+const notes = [];
+function saveNote(content, id) {
+  let obj = { content, id };
+  notes.push(obj);
+  return;
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes);
+
+function getNote(id) {
+  // your code here
+
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id == id) {
+      let m = notes[i].content;
+      return m;
+    }
+  }
+  return "error";
```

Comment:
The function should return note, and if the note does not exist we want to show an error. I think that the best way is to throw an error instead of returning a string. Replace this line with:
```js
throw new Error('note not found!');
```